From: John Crispin <john@phrozen.org>
Date: Thu, 23 May 2019 10:27:24 +0200
Subject: [PATCH V2] mac80211: fix rate reporting inside
 cfg80211_calculate_bitrate_he()

The reported rate is not scaled down correctly. After applying this patch,
the function will behave just like the v/ht equivalents.

Signed-off-by: Shashidhar Lakkavalli <slakkavalli@datto.com>
Signed-off-by: John Crispin <john@phrozen.org>
---
 net/wireless/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/net/wireless/util.c
+++ b/net/wireless/util.c
@@ -1241,7 +1241,7 @@ static u32 cfg80211_calculate_bitrate_he
 	if (rate->he_dcm)
 		result /= 2;
 
-	return result;
+	return result / 10000;
 }
 
 u32 cfg80211_calculate_bitrate(struct rate_info *rate)
